recipes/mypy-boto3-s3/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mypy-boto3-s3" %}
+{% set version = "1.17.71.post1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mypy-boto3-s3-{{ version }}.tar.gz
+  sha256: 2a82d628a1d4741d40124f8eab7b1918e3c6326319e0f0ed68070677f61f8b1a
+
+build:
+  number: 0
+  skip: true   # [py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - boto3
+    - python
+    - typing-extensions  # [py<38]
```

Comment:
This can be made noarch python if you remove the selector from this line

Reply:
Okay I'll do that!